### PR TITLE
Fix #747 by updating async SQLAlchemy OAuth example code

### LINKED ISSUE #747: SQLAlchemy example for async_find_bot out of date
https://github.com/slackapi/bolt-python/blob/71c89cca62fe28b37bb3eba6263fa91451a6601b/examples/sqlalchemy/async_oauth_app.py#L66

The function is missing `is_enterprise_install` as a parameter. Submitting a PR,

### PR DESCRIPTION
**Pull request by ntarora**
This pull request resolves #747

### CODE REVIEW

**Review by ntarora — COMMENTED** [review]
Add is_enterprise_install to example

**Comment by ntarora** [comment]
fix #747

File: examples/sqlalchemy/async_oauth_app.py [hunk 1]
```diff
@@ -63,11 +63,11 @@ async def async_save(self, installation: Installation):
                 b["client_id"] = self.client_id
                 await database.execute(self.bots.insert(), b)
 
-    async def async_find_bot(self, *, enterprise_id: Optional[str], team_id: Optional[str]) -> Optional[Bot]:
+    async def async_find_bot(self, *, enterprise_id: Optional[str], team_id: Optional[str], is_enterprise_install: Optional[bool]) -> Optional[Bot]:
```
Comment (seratch) [inline examples/sqlalchemy/async_oauth_app.py:66]: flake8 detects lint errors. Can you these lines shorter than 125 chars? 

```
Successfully installed flake8-5.0.4 mccabe-0.7.0 pycodestyle-2.9.1 pyflakes-2.5.0
examples/sqlalchemy/async_oauth_app.py:66:126: E501 line too long (148 > 125 characters)
examples/sqlalchemy/async_oauth_app.py:70:126: E501 line too long (130 > 125 characters)
Error: Process completed with exit code 1.
```

**Review by seratch — CHANGES_REQUESTED** [review]

File: examples/sqlalchemy/async_oauth_app.py (hunk 1 above)
Comment (seratch) [reply examples/sqlalchemy/async_oauth_app.py:66]: You can fix these warnings by the following changes:
```diff
diff --git a/examples/sqlalchemy/async_oauth_app.py b/examples/sqlalchemy/async_oauth_app.py
index 5ec7be0..a2c52f0 100644
--- a/examples/sqlalchemy/async_oauth_app.py
+++ b/examples/sqlalchemy/async_oauth_app.py
@@ -63,11 +63,23 @@ class AsyncSQLAlchemyInstallationStore(AsyncInstallationStore):
                 b["client_id"] = self.client_id
                 await database.execute(self.bots.insert(), b)
 
-    async def async_find_bot(self, *, enterprise_id: Optional[str], team_id: Optional[str], is_enterprise_install: Optional[bool]) -> Optional[Bot]:
+    async def async_find_bot(
+        self,
+        *,
+        enterprise_id: Optional[str],
+        team_id: Optional[str],
+        is_enterprise_install: Optional[bool],
+    ) -> Optional[Bot]:
         c = self.bots.c
         query = (
             self.bots.select()
-            .where(and_(c.enterprise_id == enterprise_id, c.team_id == team_id, c.is_enterprise_install == is_enterprise_install))
+            .where(
+                and_(
+                    c.enterprise_id == enterprise_id,
+                    c.team_id == team_id,
+                    c.is_enterprise_install == is_enterprise_install,
+                )
+            )
             .order_by(desc(c.installed_at))
             .limit(1)
         )
```

Comment (seratch) [reply examples/sqlalchemy/async_oauth_app.py:66]: @ntarora Could you push the fix if you have a chance?

**Comment by ntarora** [comment]
Odd, I did push the formatting fix on my branch. I didn’t get picked up for some reason. Should I close this PR and create a new one? On Oct 26, 2022, at 1:39 AM, Kazuhiro Sera ***@***.***> wrote:﻿
@seratch commented on this pull request.



In examples/sqlalchemy/async_oauth_app.py:
> @@ -63,11 +63,11 @@ async def async_save(self, installation: Installation):
                 b["client_id"] = self.client_id
                 await database.execute(self.bots.insert(), b)
 
-    async def async_find_bot(self, *, enterprise_id: Optional[str], team_id: Optional[str]) -> Optional[Bot]:
+    async def async_find_bot(self, *, enterprise_id: Optional[str], team_id: Optional[str], is_enterprise_install: Optional[bool]) -> Optional[Bot]:

@ntarora Could you push the fix if you have a chance?

**Comment by seratch** [comment]
Thanks for your contribution! Your changes are merged at https://github.com/slackapi/bolt-python/commit/f5b40c7830fc0fe681388be0d110054873374a8b and then I've added an additional commit to resolve the flake8 issue on my end.

**Pull request closed without merging**